README.md

```diff
@@ -116,3 +116,22 @@ Either set `HARMONY_DIR` or replace it with your Harmony project directory path.
 Harmony-py uses [GitHub
 Actions](https://github.com/nasa/harmony-py/actions) to run the Linter
 & Unit Tests. The test coverage output is saved as a build artifact.
+
+## Building and Releasing
+
+If a new version of Harmony-Py has been released then `master` should be tagged with an updated version:
+
+        $ git checkout master
+        $ git tag -a 1.2.3    # where 1.2.3 is the next version number
```

Comment:
harmony-service-lib-py prefixes its version tags with `v`.  Can we be consistent with that?